trust_stores/openjdk.yaml

```diff
@@ -0,0 +1,192 @@
+platform: OPENJDK
+version: null
```

Comment:
I didn't set the version, as it is the latest in the source repository - there will rarely be a meaningful tag matching an OpenJDK release version.

We could use the revision hash, but that wouldn't say much. It would be exact, but speaking less than the date.

It is identical to the Mozilla store, fetched from the repository - the version is null there as well.